submodules/LIB/scripts/iob_module.py

```diff
@@ -419,8 +421,8 @@ def _generate_hw(cls, mkregs_obj, reg_table):
     @classmethod
     def _generate_sw(cls, mkregs_obj, reg_table):
         """Generate common software files"""
-        os.makedirs(cls.build_dir + "/software/src", exist_ok=True)
         if cls.regs:
+            os.makedirs(cls.build_dir + "/software/src", exist_ok=True)
```

Comment:
```suggestion
        os.makedirs(cls.build_dir + "/software/src", exist_ok=True)
        if cls.regs:
```

Reply:
The problem with that is that every module will create a _conf.h file, adding dozens of unused files